frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -75,14 +85,31 @@ export class PieceEditor implements OnInit {
     this.pieceAPI.getAllPieces();
   }
 
+  private buildRenderingList() {
+    this.renderingItems = this.workingPieces.map((wpiece, wpieceIx) => {
+      const missingChidren = wpiece.type === 'SEGMENT' ? false : wpiece.positions.length === 0;
```

Comment:
`missingChildren` sounds like I would expect it to contain a list of "child" items, not a boolean.

How about something like `hasNoChildren`

Or even better, invert the negation to reduce complexity and make it
`const hasChildren = ...`

Reply:
Done. I prefer hasNoChildren to hasChildren. We are looking for the pieces/segments that have no children.